cli/template.go

```diff
@@ -54,6 +58,38 @@ func fixEnumDefaultValue(node map[string]interface{}) {
 	}
 }
 
+func getType(input interface{}) (valueType string, nullable bool, err error) {
+	valueType, _ = input.(string)
+	if valueType != "" {
+		return
+	}
+
+	types, _ := input.([]interface{})
+	for _, item := range types {
+		item := item.(string)
+		if item == "null" {
+			nullable = true
+		} else {
+			if valueType != "" {
+				return "", false, fmt.Errorf("more then one type")
```

Comment:
You don't need a formatting function when there is no formatting done. Which gave me an idea: `fmt.Errorf("more than one type in %+v", input)`. WDYT?

Reply:
Good idea, done